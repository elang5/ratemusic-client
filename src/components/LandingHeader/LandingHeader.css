.landing-header {
  background: url('https://images.unsplash.com/3/www.madebyvadim.com.jpg?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1661&q=80') center;
  background-size: cover;
  background-repeat: no-repeat;
  padding: 4rem 2rem;
}

.header-title {
  text-align: right;
}
