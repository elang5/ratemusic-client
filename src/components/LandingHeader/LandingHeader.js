import React from 'react'
import './LandingHeader.css'

export default function LandingHeader() {
  return (
    <header className="landing-header">
      <h1 className="header-title">Rate Music</h1>
    </header>
  )
}
