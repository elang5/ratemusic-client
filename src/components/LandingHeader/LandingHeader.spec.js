import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import LandingHeader from './LandingHeader'

describe(`LandingHeader Component`, () => {

  it(`renders the LandingHeader component`, () => {
    const wrapper = shallow(<LandingHeader />)
    expect(toJson(wrapper)).toMatchSnapshot()
  })
})