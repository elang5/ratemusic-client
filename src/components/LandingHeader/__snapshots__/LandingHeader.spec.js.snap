// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingHeader Component renders the LandingHeader component 1`] = `
<header
  className="landing-header"
>
  <h1
    className="header-title"
  >
    Rate Music
  </h1>
</header>
`;
